packages/react/src/progress-indicator/ProgressIndicatorItem.tsx

```diff
@@ -44,6 +44,11 @@ export const ProgressIndicatorItem = ({
 		);
 	}
 
+	const hasSubLevel = isActive && items?.some((item) => item.isActive);
+	const subLevelItem = hasSubLevel
+		? items?.find((item) => item.isActive)
+		: undefined;
```

Comment:
It doesn't look like `hasSubLevel` is being used after line 48 and the `some` performs the same check as the `find`, so you could probably roll them into one variable:

```suggestion
	const subLevelItem = isActive
		? items?.find((item) => item.isActive)
		: undefined;
```